content/95-ismms-himc-appendix.md

```diff
@@ -1,4 +1,4 @@
-<!-- Appendix source: https://github.com/ismms-himc/covid-19_sinai_reviews/tree/b8d8df55b4f2416da4ca3b7c765c8fff48acf5e3 -->
+<!-- Appendix source: https://github.com/ismms-himc/covid-19_sinai_reviews/tree/05d387bf10b829dad6bedb4bf313686750b78cc9 -->
```

Comment:
@rando2 you don't have to hash anything yourself.  My script update in #178 updates this line in the appendix when you re-run the script.  It gets the latest commit from https://github.com/ismms-himc/covid-19_sinai_reviews and uses that SHA-1 hash to version the appendix.  It looks like everything is working as intended.

Reply:
Awesome! Thanks @agitter!